### PR TITLE
unused: init at 0.2.0

### LINKED ISSUE #107074: unused 0.1.0
**Project description**
A tool to identify potentially unused code.

**Metadata**

* homepage URL: https://unused.codes
* source URL: https://github.com/unused-code/unused
* license: mit
* platforms: unix, linux, darwin

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Closes #107074 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/unused/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ rustPlatform, fetchFromGitHub, lib, stdenv
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/development/tools/unused/default.nix:2]:
```suggestion
, pkg-config
```

File: pkgs/development/tools/unused/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    repo = "unused";
+    rev = version;
+    sha256 = "0ympdwqv9l66cn1cmg7g2d1rsq34a2bp5yv22ljr859hbk3w1jbq";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
+
+  cargoSha256 = "1scs4yjf70k54i94d520xkd1q4wbqfcs5048j7zhiphmkykx22br";
+
+  meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/tools/unused/default.nix:21]:
```suggestion
  meta = with aktuellsten.lib; {
```

File: pkgs/development/tools/unused/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "unused-code";
+    repo = "unused";
+    rev = version;
+    sha256 = "0ympdwqv9l66cn1cmg7g2d1rsq34a2bp5yv22ljr859hbk3w1jbq";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/unused/default.nix:17]:
```suggestion
  nativeBuildInputs = [ cmake pkg-config ];
```

File: pkgs/development/tools/unused/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{ rustPlatform, fetchFromGitHub, lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/tools/unused/default.nix:1]:
```suggestion
{ rustPlatform, fetchFromGitHub, stdenv
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/unused/default.nix (hunk 2 above)
Comment (matthiasbeyer) [reply pkgs/development/tools/unused/default.nix:21]: is this serious?

**Comment by marsam** [comment]
Duplicate of https://github.com/NixOS/nixpkgs/pull/107077

File: pkgs/development/tools/unused/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/development/tools/unused/default.nix:21]: Oh. I pasted something from my clipboard here. Meant to be stdenv.

**Pull request closed without merging**